src/Sunburst.ts

```diff
@@ -664,15 +672,24 @@ export class Sunburst implements IVisual {
     }
 
     private setPercentageLabelPosition(width: number, canDisplayCategory: boolean): void {
-        const labelSize: number = this.settings.group.selectedCategory.font.fontSize.value * Sunburst.PercentageFontSizeMultiplier;
+        const percentageLabelSettings = this.settings.centralLabel.percentageLabel;
```

Comment:
I would suggest: `const fontSettings = this.settings.centralLabel.percentageLabel.font`
just for less code